airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -88,7 +90,9 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
         Workflow.getVersion(AUTO_DETECT_SCHEMA_TAG, Workflow.DEFAULT_VERSION,
             AUTO_DETECT_SCHEMA_VERSION);
 
-    if (autoDetectSchemaVersion >= AUTO_DETECT_SCHEMA_VERSION) {
+    final int discoveryOptionsVersion = Workflow.getVersion(DISCOVERY_OPTIONS_TAG, Workflow.DEFAULT_VERSION, DISCOVER_OPTIONS_VERSION);
```

Comment:
@benmoriceau I added a new version here, but still seeing test failures

Reply:
I think that we should have 2 `private RefreshSchemaActivity refreshSchemaActivity;` with the different options. Then you'll need to use the new version to call one or the other activity.

Reply:
Just tried this, but I'm still seeing that test failure

Reply:
just pushed up my changes - is that what you meant?

Reply:
The issue is in the stack trace of the failing test, The bean is not properly defined, I'm trying to make a fix.

Reply:
I made a fix for the workflow replayer, another test need to be updated.